FeatureDetect/root/examples/test.cpp

```diff
@@ -0,0 +1,6 @@
+#include "../source/featurelib.h"
+
+int main() {
+featurelib instance();
```

Comment:
Is it really ok for the linter that this line has no indentation?

Reply:
paul@paul-VivoBook-ASUSLaptop-X515EA-R565EA:~/CLionProjects/FeatureDetectAndroid/examples$ cpplint test.cpp
test.cpp:0:  No copyright message found.  You should have a line: "Copyright [year] <Copyright Owner>"  [legal/copyright] [5]
Done processing test.cpp
Total errors found: 1
(base) paul@paul-VivoBook-ASUSLaptop-X515EA-R565EA:~/CLionProjects/FeatureDetectAndroid/examples$ cat test.cpp
#include "../source/featurelib.h"

int main() {
featurelib instance();
    return 0;

Reply:
yep

Reply:
Well, then fix it by hand as it doesn't comply with [the code style](https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions)